ProcessMaker/EncryptedData/EncryptedDataInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace ProcessMaker\EncryptedData;
+
+interface EncryptedDataInterface
+{
+    public function encryptText(string $plainText, string $iv = null, string $key = null): string;
```

Comment:
Please add the documentation of the methods, explain when the $iv and $key can be optional, and when not.

Reply:
done